pkg/cluster/connectionsManager.go

```diff
@@ -0,0 +1,68 @@
+/**
+ * Handles specific operations for Kafka Connection resource
+ */
+package cluster
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/redhat-developer/app-services-cli/pkg/cluster/constants/kafka"
+	"github.com/redhat-developer/app-services-cli/pkg/cluster/constants/serviceregistry"
+	kafkamgmtclient "github.com/redhat-developer/app-services-sdk-go/kafkamgmt/apiv1/client"
+	srsmgmtv1 "github.com/redhat-developer/app-services-sdk-go/registrymgmt/apiv1/client"
+)
+
+// IsKCInstalledOnCluster checks the cluster to see if a KafkaConnection CRD is installed
+func IsKCInstalledOnCluster(ctx context.Context, c *KubernetesCluster) (bool, error) {
```

Comment:
function probably should be renamed to 
`IsResourceInstalledonCluster` and accept resource from constants and then possibly moved to utils?

Reply:
Noticed we would need to check for SBO as well.

Reply:
Yes! :D

Reply:
Facing issues clubbing them to utils. But this is indeed no place to be keeping them.